cmd/tink-worker/main.go

```diff
@@ -19,31 +22,32 @@ func main() {
 		fmt.Fprintln(os.Stderr, "Error:", err)
 		os.Exit(1)
 	}
+	logger := getLogger(flagEnvSettings.LogLevel)
 
 	// Randomized sleep function. Used when retrying failed connections to tink server.
 	sleepFn := func() {
 		rand.Seed(time.Now().UnixNano())
 		s := rand.Intn(120) + 1 // 2 minutes (120 seconds), plus one second to avoid a possible zero sleep time
-		fmt.Fprintf(os.Stderr, "Sleeping %d seconds before attempting to re-connect...\n", s)
+		logger.Info("Sleeping before attempting to re-connect...", "sleep time", s)
 		time.Sleep(time.Duration(s) * time.Second)
 	}
 
 	// Here we retry failed connections to tink-server using a randomized interval between attempts.
 	// tink-worker is a daemon process, so we do not exit here.
 	var conn *grpc.ClientConn
 	for {
-		fmt.Fprintf(os.Stderr, "Obtaining tink server creds from %s...\n", flagEnvSettings.TinkServerURL)
+		logger.V(0).Info("Obtaining tink server creds from Tink-Server URL", "Tink-Server URL", flagEnvSettings.TinkServerURL)
 		creds, err := tink.ObtainServerCreds(flagEnvSettings.TinkServerURL)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error obtaining server creds from %s: %v\n", flagEnvSettings.TinkServerURL, err)
+			logger.V(2).Error(err, "Error obtaining server creds from Tink-Server URL", "Tink-Server URL", flagEnvSettings.TinkServerURL)
```

Comment:
What's the thinking around using v level 2 here?

Reply:
The intent was to display only an error message.